### DISCUSSION TITLE
Child Template

### DISCUSSION THREAD

**Discussion by ghost** [joomla/joomla-cms#32442] [Q&A] [answered]
Is there a documentation how to create a child template.

I meam how to create it inside joomla's env like wrote unter "Front End step 5" in https://github.com/joomla/joomla-cms/pull/30192

**Comment by dgrammatiko** [comment, marked as answer]
> I meam how to create it inside joomla's env like wrote unter "Front End step 5" in

At the moment this feature is not exposed in the GUI. There is a PR: https://github.com/joomla/joomla-cms/pull/30359 and I'm working on it at the moment so hopefully, in the coming days there will be a PR that people could test

**Reply by ghost** [reply]
Thanks

**Reply by andiamine** [reply]
amazing thanks

**Comment by ghost** [comment]
I just wanted to test the creation of a child template in joomla 4. I have installed the nighly build of 4.1 and cassy.I have attached my system information.
For me, neither under Cassiopiea nor under Cassy does the Child Template button appear.  It is still called Copy Template. 
What am I doing wrong?




## System Information

php: Linux php80_mysql 5.11.0-43-generic #47~20.04.2-Ubuntu SMP Mon Dec 13 11:06:56 UTC 2021 x86_64
dbserver: mysql
dbversion: 5.7.34
dbcollation: latin1_swedish_ci
dbconnectioncollation: utf8mb4_general_ci
dbconnectionencryption: 
dbconnencryptsupported: true
phpversion: 8.0.8
server: Apache/2.4.41 (Ubuntu)
sapi_name: apache2handler
version: Joomla! 4.1.0-dev Development [ Kuamini ] 21-December-2021 18:00 GMT
useragent: Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:95.0) Gecko/20100101 Firefox/95.0

...


cassy

name: cassy
type: template
state: Enabled
author: Joomla! Project
version: 1.0
creationDate: February 2017
authorUrl:

**Reply by dgrammatiko** [reply]
The nightly doesn\'t have the changes for the Child Templates. Go to https://developer.joomla.org/nightly-builds.html and select the last option there `Joomla! 4.1`. You won't need Cassy, Cassiopeia will work fine...

**Comment by ghost** [comment]
Thank you @dgrammatiko 
I have installed that. In the System Informations you can see:
`version: Joomla! 4.1.0-dev Development [ Kuamini ] 21-December-2021 18:00 GMT`

**Reply by dgrammatiko** [reply]
And you tried to create a child from `cassiopeia` or `atum` and didn't work?

**Reply by ghost** [reply]
I tried it with Cassipeia, but I did not see a "Child Template" button. Then I installed Cassy. I do see it either. 
Now I tried it wit Atum. Here I see the "Child Template" button.
Where can my mistake be?
![1](https://user-images.githubusercontent.com/73401898/147701864-65a9e1aa-ac65-4356-baba-98f9bbf505be.png)
![2](https://user-images.githubusercontent.com/73401898/147701867-d03cf28c-0a2f-4b53-8cba-5f2f6b1ecb0e.png)

**Reply by dgrammatiko** [reply]
Is this a clean installation or an update?

**Reply by ghost** [reply]
Yes it was a clean installation. 
I downloaded and installed a J4.1 Nighly again today. Now it also works with Cassiopeia. No idea why it yesterday did not work.
